Marlin/Marlin_main.cpp

```diff
@@ -8248,7 +8266,6 @@ inline void gcode_M205() {
   inline void gcode_M665() {
     if (parser.seen('H')) {
       home_offset[Z_AXIS] = parser.value_linear_units() - DELTA_HEIGHT;
-      current_position[Z_AXIS] += parser.value_linear_units() - DELTA_HEIGHT - home_offset[Z_AXIS];
```

Comment:
@LVD-AC I realized this line does nothing. Probably I left it behind from a debugging session.